lib/commands/sdk.js

```diff
@@ -369,91 +319,74 @@ SdkSubcommands.select = {
 	fn: callbackify(select)
 };
 
+function sortVersion(a, b) {
+	const [ _, amajor, aminor, apatch, atag ] = a.name.toLowerCase().match(versionRegExp);
```

Comment:
No need to have the `_` here unless you're doing it to make it obvious
```suggestion
	const [ , amajor, aminor, apatch, atag ] = a.name.toLowerCase().match(versionRegExp);
```

Reply:
Fixed.